app/routes.test.js

```diff
@@ -216,17 +216,48 @@ describe('routes', () => {
   });
 
   describe('POST /solutions/capabilities-selector', () => {
-    it('should return the correct status and text if there is no error', () => {
+    it('should return the correct status and text if there is no error and one capability selected', () => {
+      solutionListPageContext.getSolutionsForSelectedCapabilities = jest.fn()
+        .mockImplementation(() => Promise.resolve(mockFilteredSolutions));
+      const app = new App().createApp();
+      app.use('/', routes);
+      return request(app)
+        .post('/solutions/capabilities-selector')
+        .send({ capabilities: 'C1' })
+        .expect(302)
+        .then((res) => {
+          expect(res.redirect).toEqual(true);
+          expect(res.text).toEqual('Found. Redirecting to /solutions/capabilities-selector.C1');
```

Comment:
I don't get this? is the response we return on the page?

Reply:
it's just a key added to the res object - I was looking for where I could find the redirect url to test and found this which has it in

Reply:
ok so this is not the actual body of the res?

Reply:
I have no idea what you mean but it is put on the res by the redirect so there is not much we can do about it if you don't like it 😄

Reply:
haha why so defensive. I like it! I understand why now. The result of the unit test is the redirect rather an actual page to render. That's what confused me

Reply:
yeah I just wanted to access the redirect url and this was the first place I found it